backend/persistent/QueryScheduler.js

```diff
@@ -381,11 +387,20 @@ class QueryScheduler {
                 */
                 throw new Error(`MetadataError: ${e.message}`);
             });
-        }).then((res) => {
+        }).then(() => {
             Logger.log(`Request to Plotly for creating a grid took ${process.hrtime(startTime)[0]} seconds`, 2);
-            return res.json().then(() => {
-                Logger.log(`Grid ${fid} has been updated.`, 2);
-            });
+            Logger.log(`Grid ${fid} has been updated.`, 2);
+
+            startTime = process.hrtime();
+
+            // fetch updated grid column for returning
+            return getGridColumn(fid, requestor);
+        }).then((res) => {
+            Logger.log(`Request to Plotly for fetching updated grid took ${process.hrtime(startTime)[0]} seconds`, 2);
+            if (res.status !== 200) {
+              return res.text();
```

Comment:
this is silencing the error we currently see in master

Reply:
Interesting 🤔This wasn't added until https://github.com/plotly/falcon-sql-client/pull/511/commits/dc4091249e48a10db12068abe1838900f36437a5, but that test was still passing beforehand. Any idea how that is @n-riesco?

Reply:
@briandennis After more debugging, I have a better idea of what's happening.

Currently, master is failing because we get `400 Bad Request` in https://github.com/plotly/falcon-sql-client/pull/511/files#diff-a353f58a9179598892ca8c2a867f4e2dR309

In master, this causes an exception in https://github.com/plotly/falcon-sql-client/pull/511/files#diff-a353f58a9179598892ca8c2a867f4e2dL384 because `res` in `undefined`.

In this PR, execution moves to https://github.com/plotly/falcon-sql-client/pull/511/files#diff-a353f58a9179598892ca8c2a867f4e2dR390 as if the grid hadn't been deleted.